src/classiceditorui.js

```diff
@@ -88,6 +99,29 @@ export default class ClassicEditorUI {
 				}
 
 				return Promise.all( promises );
+			} )
+			.then( () => {
+				// Listen on the keystrokes from the main UI.
+				this.keystrokes.listenTo( this.view.element );
```

Comment:
Because to start `this.keystrokes.listenTo( view._bodyCollectionContainer );`, `EditorUIView` must do `.then( () => this._renderBodyCollection() )`, which happens on `view.init()`, which in turn happens on `ui.init()`.